### ISSUE TITLE
Irc -> Chosen Channel

### ISSUE THREAD

**Issue by mashlol** [Monstercraft/MonsterIRC#122]
It would be nice if it were possible with HeroChat to allow chat from an IRC channel to go to a certain HeroChat channel, and from a HeroChat channel to go to a certain IRC channel.  Currently it seems only possible to go from one HeroChat channel back and forth between a single IRC channel.  More customization there would be great.

For example:

Lets say I have two herochat channels, Global, and IRC.  I want to connect global so it is mirrored to #serverglobal, but I want chat from #serverglobal to go to the HeroChat channel called IRC.  Doesn't seem possible as is.

Basically I would like to be able to specify where incoming chat goes from the IRC channel, once it gets to the server.
